tools/http2_interop/doc.go

```diff
@@ -1,3 +1,17 @@
+// Copyright 2019 gRPC authors.
```

Comment:
The new format is `Copyright 2019 The gRPC Authors`
as seen e.g. here: https://github.com/grpc/grpc/blob/906c12568e9bdf1d8306a2f2ab45efe8beb43f74/src/csharp/Grpc.Core.Api/Properties/AssemblyInfo.cs#L3

Reply:
Thank you!